debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/SourceInfo.java

```diff
@@ -162,6 +162,13 @@ Position position() {
         return position;
     }
 
+    /**
+     * @return server id
+     */
+    public String serverId() {
```

Comment:
This method is needed to implement the newly added `io.debezium.connector.mongodb.MongoDbOffsetContext #getReplicaSetOffsetContext`. It can be removed as soon as the connector is refactored to use the multi-partition API ([DBZ-3703)](https://issues.redhat.com/browse/DBZ-3703).